pkg/controller/kfdef/kfdef_controller.go

```diff
@@ -35,6 +42,11 @@ const (
 	finalizer = "kfdef-finalizer.kfdef.apps.kubeflow.org"
 	// finalizerMaxRetries defines the maximum number of attempts to add finalizers.
 	finalizerMaxRetries = 10
+	// deleteConfigMapLabel is the label for configMap used to trigger operator uninstall
+	// TODO: Label should be updated if addon name changes
+	deleteConfigMapLabel = "api.openshift.com/addon-managed-odh-delete"
+	// rhodsGeneratedNamespaceLabel is the label added to all the namespaces genereated by odh-deployer
+	rhodsGeneratedNamespaceLabel = "opendatahub.io/generated-namespace"
```

Comment:
I think we would also want the variable name to be more like "odhGeneratedNamespaceLabel"